packages/cli/yaml/yaml-schema/src/schemas/UndiscriminatedUnionSchema.ts

```diff
@@ -0,0 +1,17 @@
+import { z } from "zod";
+import { BaseTypeDeclarationSchema } from "./BaseTypeDeclarationSchema";
+import { WithDocsSchema } from "./WithDocsSchema";
+
+export const UndiscriminatedUnionSchema = BaseTypeDeclarationSchema.extend({
+    union: z.array(
+        z.union([
+            z.string(),
+            WithDocsSchema.extend({
+                type: z.string(),
+            }),
+        ])
+    ),
+    discriminated: z.literal(false),
```

Comment:
on the other union could see you doing `discriminated: z.optional(z.literal(true))`

I think it's more common to have optional booleans default to false, but in this case it's defaulting to true. any ideas for a key that defaults to false? nbd if not.

Reply:
`undiscriminated` ?  feels more complex than now